### PR TITLE
Added module for plantuml

### PR DESCRIPTION
**Pull request by wanderer**
This adds a modules for [weirongxu/plantuml-previewer](https://github.com/weirongxu/plantuml-previewer.vim)

### CODE REVIEW

File: modules/plantuml/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      vim.startPlugins = [
+        "plantuml-previewer"
+        pkgs.vimPlugins.plantuml-syntax
+        pkgs.vimPlugins.open-browser
+      ];
+
+      vim.configRC.plantuml = nvim.dag.entryAnywhere ''
+        au filetype plantuml let g:plantuml_previewer#viewer_path = "/tmp/plantuml"
+        au filetype plantuml let g:plantuml_previewer#java_path = "${pkgs.openjdk}/bin/java"
```
Comment (jordanisaacs) [inline modules/plantuml/default.nix]: Can you make two options for setting the packages. eg `vim.plantuml.pkgs.{plantuml,openjdk}` and they can just default to `pkgs.openjdk` etc.

Comment (wanderer) [reply modules/plantuml/default.nix]: yep, done!

**Comment by wanderer** [comment]
@jordanisaacs let me know if you want that last commit squashed

**Comment by jordanisaacs** [comment]
If you could squash it that would be great. Also if you could address the vim plugins as inputs which I mentioned in a comments. Plus, a quick line in the release doc about the new module. Feel free to include your name/profile if you want. Otherwise lgtm! @wanderer

**Comment by jordanisaacs** [comment]
The release file: https://github.com/jordanisaacs/neovim-flake/blob/main/docs/release-notes/rl-0.1.adoc

**Comment by wanderer** [comment]
@jordanisaacs just to clarify on this comment

> . Also if you could address the vim plugins as inputs which I mentioned in a comments.

I should make `pkgs.vimPlugins.plantuml-syntax` and `pkgs.vimPlugins.open-browser` inputs to the flake instead of using the versions from nixpkgs?

**Comment by jordanisaacs** [comment]
@wanderer yep exactly.

**Comment by wanderer** [comment]
@jordanisaacs I believe everything is done now.

**Comment by jordanisaacs** [comment]
lgtm thanks!

**Pull request merged**